Examcc/src/exo3/Dictionary.java

```diff
@@ -0,0 +1,23 @@
+package exo3;
+
+import java.util.ArrayList;
+
+public class Dictionary {
+	ArrayList<Word> al= new ArrayList<Word>();
+
+	public void add (Word W) {
+		al.add(W);
+	}
+
+	public Word find (String id) {
+		for(Word word : al) {
```

Comment:
If you use a map you can speed up find and do it with **one single** instruction: `return al.get(id)`